packages/forma-36-react-components/src/components/CheckboxField/CheckboxField.stories.tsx

```diff
@@ -1,97 +1,116 @@
 import React, { useState } from 'react';
-import { storiesOf } from '@storybook/react';
-import { text, boolean } from '@storybook/addon-knobs';
-import { action } from '@storybook/addon-actions';
 
 import FieldGroup from '../Form/FieldGroup';
-import CheckboxField from './CheckboxField';
+import CheckboxField, { CheckboxFieldProps } from './CheckboxField';
 import Flex from '../Flex/Flex';
 import SectionHeading from '../Typography/SectionHeading';
 
-function DefaultStory() {
+export default {
+  title: 'Components/CheckboxField',
+  component: CheckboxField,
+  parameters: {
+    propTypes: [CheckboxField['__docgenInfo']],
+  },
+  argTypes: {
+    className: { control: { disable: true } },
+    testId: { control: { disable: true } },
+    checked: { control: { disable: false } },
+    required: { control: { disable: false } },
+    disabled: { control: { disable: false } },
+    labelIsLight: { control: { disable: false } },
+  },
+};
+
+const BasicStory = (args: CheckboxFieldProps) => {
   const [optionOne, setOptionOne] = useState(false);
   const [optionTwo, setOptionTwo] = useState(false);
 
   return (
     <FieldGroup>
       <CheckboxField
-        labelText={text('labelText', 'Option 1')}
-        helpText={text('helpText', 'This is a helptext')}
-        validationMessage={text('validationMessage', undefined)}
-        disabled={boolean('disabled', false)}
-        name="someOption"
+        id="Checkbox"
+        labelText={args['labelText']}
+        name={args.name}
+        helpText={args.helpText}
+        validationMessage={args.validationMessage}
```

Comment:
Same thing here, you just need to pass `{...args}` and storybook will do the rest